build.sh

```diff
@@ -347,19 +347,20 @@ isMSBuildOnNETCoreSupported()
         return
     fi
 
+    if [[ ("$__HostOS" == "Linux") && ("$__HostArch" == "x64" || "$__HostArch" == "arm64") ]]; then
```

Comment:
We also should support armhf. Please modify this to:

> if [[ ("${__HostOs}" == "Linux") && ("${__HostArch}" == "x64" || "${__HostArch}" == "arm" || "${__HostArch}" == "arm64") ]]; then

Reply:
Done.